macos-base.pkr.hcl

```diff
@@ -129,6 +129,7 @@ build {
             "rbenv install 3.1.4",
             "rbenv global 3.1.4",
             "gem install bundler",
+            "gem install cocoapods",
```

Comment:
I would definitely move that also to the xcode image and probably to flutter section as otherwise do not need it, do we?

Reply:
okay, np. e14e979f6ef1c878c8579a1bc5a8cda0bbd23373